### PR TITLE
[Nano] Add save/load fundamental examples for optimized Pytorch model

### PR DESCRIPTION
**Pull request by hjzin**
## Description

Add save/load fundamental examples for optimized Pytorch model.
### 1. Why the change?

related issue: #5773 

### 2. User API changes

N/A
### 3. Summary of the change 

Add save/load fundamental examples for optimized Pytorch model using openvino and onnxruntime as accelerator.
### 4. How to test?
- [ ] Unit test

### CODE REVIEW

**Comment by rnwang04** [comment]
Maybe use InferenceOptimizer instead of Trainer here.

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ort_model = InferenceOptimizer.trace(model_ft,
+                                         accelerator="onnxruntime",
+                                         input_sample=torch.rand(1, 3, 224, 224))
+
+    y_hat = ort_model(x)
+    predictions = y_hat.argmax(dim=1)
+    print(predictions)
+
+    # Save Optimized Model
+    from bigdl.nano.pytorch import Trainer
```
Comment (rnwang04) [inline python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:43]: No need for `import Trainer`.
Now InferenceOptimizer also provide save / load function. You can just import InferenceOptimizer and use InferenceOptimizer.save / InferenceOptimizer.load here.

Comment (jason-dai) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:43]: Explain how the user can also retrieve the standard model (ONNX or OpenVINO) files

Comment (TheaperDeng) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:43]: Need explaination about the files in saving dir (one is a onnx/openvino ckpt for general use, the other one is a meta information illustrates the precision and other information about the general ckpt.)

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# ```
+
+import torch
+from torchvision.models import resnet18
+
+if __name__ == "__main__":
+    model_ft = resnet18(pretrained=True)
+
+    x = torch.rand(2, 3, 224, 224)
+    model_ft.eval()
```
Comment (TheaperDeng) [inline python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:30]: no need to switch to eval mode

Comment (rnwang04) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:30]: At present， it seems that we don't set `model.eval()` in openvino and onnxruntime.

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py (hunk 1 above)
Comment (hjzin) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py]: Added some explanation for saved model files

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py (hunk 2 above)
Comment (hjzin) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py]: > At present, it seems that we don't set `model.eval()` in openvino and onnxruntime.

So you think maybe we could add `model.eval` here?

Comment (rnwang04) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py]: Yeah, i think could add here.  Maybe we should add `model.eval()` in `trace` and `quantize` later.

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    y_hat = ort_model(x)
+    predictions = y_hat.argmax(dim=1)
+    print(predictions)
+
+    # Save Optimized Model
+    # The saved model files will be saved at "./optimized_model_ort" directory
+    # There are 2 files in optimized_model_ort:
+    #   nano_model_meta.yml: meta information of the saved model checkpoint
+    #   onnx_saved_model.onnx: model checkpoint for general use, describes model structure
```
Comment (TheaperDeng) [inline python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py:46]: maybe add "Users only need to take this file for further usage."

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_openvino.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    y_hat = ov_model(x)
+    predictions = y_hat.argmax(dim=1)
+    print(predictions)
+
+    # Save Optimized Model
+    # The model files will be saved at "./optimized_model_ov" directory
+    # There are 3 files in optimized_model_ov:
+    #   nano_model_meta.yml: meta information of the saved model checkpoint
+    #   ov_saved_model.bin: contains the weights and biases binary data of model
+    #   ov_saved_model.xml: model checkpoint for general use, describes model structure
```
Comment (TheaperDeng) [inline python/nano/tutorial/inference/pytorch/pytorch_save_and_load_openvino.py:47]: maybe add "Users only need to take last 2 files for further usage."

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

File: python/nano/tutorial/inference/pytorch/pytorch_save_and_load_openvino.py (hunk 4 above)
Comment (hjzin) [reply python/nano/tutorial/inference/pytorch/pytorch_save_and_load_openvino.py:47]: done

**Review by TheaperDeng — APPROVED** [review]
LGTM
